src/app/youtube/services/sortbar-manager.service.ts

```diff
@@ -27,9 +27,12 @@ export class SortbarManagerService {
     keywords: '',
   }
 
+  options$ = new Observable<ISortOptions>()
+
   private options$$ = new BehaviorSubject(this.sortOptions)
 
   constructor() {
+    this.sortVisibility$ = this.sortVisibility$$.asObservable()
     this.options$ = this.options$$.asObservable()
```

Comment:
you can assign it to the field right away
instead of
```

  options$ = new Observable<ISortOptions>()

  constructor() {
    this.options$ = this.options$$.asObservable()
  }
```
just 
```
  options$ = this.options$$.asObservable()

```

Reply:
fixed, nice & clean solution